src/lambdaconnect_model/core.clj

```diff
@@ -44,6 +45,50 @@
   [path entities-by-name]
   (scoping/read-pull-scoping-edn path entities-by-name))
 
+(defn get-minim-scoping-sets 
+  "Given a validated scoping
+   returns map of sets which has tags as key and each set keeps
+   tags required for scoping that tag"
+  [validated-scoping]
+  (single/get-minimum-scoping-sets validated-scoping))
+
+(defn scope-selected-tags-with-tree
+  "Takes a config map, a snapshot, a user object from DB, entities-by-name, parsed (and validated) EDN of rules, map of sets indiacting which tags must be scoped per tag and a set of deisred tags.
+   It is advised to calculacte scoping sets once and pass the result.
+  A typical invocation looks like this: 
+  (scope-single-tag (d/db db/conn) user entities-by-name validated-scope scoping-sets :RARestaurant.ofOwner)))
```

Comment:
Niżej piszesz, że ostatni argument to zbiór więc chyba powinno być `#{:RARestaurant.ofOwner}`